discovery-provider/src/utils/ipfs_lib.py

```diff
@@ -186,7 +186,7 @@ def query_ipfs_metadata_json(self, gateway_ipfs_urls, default_metadata_fields):
         return formatted_json
 
     def get_metadata_from_gateway(
-        self, multihash, default_metadata_fields, user_replica_set=None
+        self, multihash, default_metadata_fields, user_replica_set: str = None
```

Comment:
```suggestion
        self, multihash, default_metadata_fields, user_cnode_endpoints: str = None
```
maybe?

Reply:
replica set is actually the correct name, thats consistent with the name of the contract, we just need to standardize this